### PR TITLE
travis: use ethereal.

### PR DESCRIPTION
**Pull request by maxice8**
- Requires oficial images

@bobertlo

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,19 +17,24 @@ env:
...
-    - common/travis/fetch_upstream.sh
-    - common/travis/changed_templates.sh
+    - docker exec -t void hostrepo/common/travis/set_mirror.sh
+    - docker exec -t void hostrepo/common/travis/prepare.sh
+    - docker exec -t void hostrepo/common/travis/fetch_upstream.sh
+    - docker exec -t void hostrepo/common/travis/changed_templates.sh
+    - common/travis/fetch-xtools.sh
     - common/travis/xlint.sh
-    - common/travis/bootstrap.sh $BOOTSTRAP
+    - docker exec -t void hostrepo/common/travis/bootstrap.sh $BOOTSTRAP
```
Comment (bobertlo) [inline .travis.yml]: argument not used

**Comment by bobertlo** [comment]
@maxice8 I was thinking about this while driving today, and wouldn't it be nice if all the things that have to happen inside the docker container are inside one script, to really simplify home use of this feature?

Basically one script that travis runs to process the matrix and pull the correct bootstrap image, and run the inner script with arguments all in one docker command. It would really only need to pass BOOTSTRAP and ARCH to the outer script, and ARCH to the inner script.

**Comment by maxice8** [comment]
No need, only arch specific stuff we need is deciding which docker image we will pull.

**Comment by the-maldridge** [comment]
Thanks for everyone's patience.  There are now masterdir images up on hub.docker.com, please test with these "official" images and confirm correct operation.  There is no :latest, it is my deliberate intention that we will commit the version change to the travis configuration.

**Comment by maxice8** [comment]
Updated:

- Added common/travis/docker.sh that will pull and run an official Void Linux docker image depending on the bootstrap arch.

**Pull request closed without merging**